dao/seeds/source_types.yml

```diff
@@ -385,20 +385,20 @@ rh-marketplace:
   vendor: Red Hat
   schema:
     authentication:
-    - type: token
-      name: API Token
-      fields:
-      - component: text-field
-        name: authentication.authtype
-        hideField: true
-        initializeOnMount: true
-        initialValue: token
-      - component: text-field
-        name: authentication.password
-        label: API Token
-        isRequired: true
-        validate:
-        - type: required
+      - type: marketplace-token
+        name: API Token
+        fields:
+        - component: text-field
+          name: authentication.authtype
+          hideField: true
+          initializeOnMount: true
+          initialValue: token
+        - component: text-field
+          name: authentication.password
+          label: API Token
+          isRequired: true
+          validate:
+          - type: required
```

Comment:
I _think_ these would need to have the 2 extra spaces removed - that way it's an array of objects.

Reply:
Right, thanks!